python/cuml/common/array_descriptor.py

```diff
@@ -54,8 +54,12 @@ class CumlArrayDescriptor():
     Python descriptor object to control getting/setting `CumlArray` attributes
     on `Base` objects. See the Estimator Guide for an in depth guide.
     """
+    def __init__(self, order='K'):
+        self.order = order
+
     def __set_name__(self, owner, name):
         self.name = name
+        setattr(owner, name + '_order', self.order)
```

Comment:
It seems like this is just setting an incorrect order label on the underlying array? Is there some other way that the order is actually getting changed?

Reply:
Here, `owner` is the estimator that owns the `CumlArrayDescriptor`, name is the name of the `CumlArrayDescriptor` attribute and `self.order` is the order received during the `CumlArrayDescriptor` construction. the estimator will now have an attribute <name>_order that specifies the order we would like that array to be with when it lands in the `CumlArrayDescriptor`. This implementation could probably be improved, but that is a working solution for the moment.

Reply:
Aha! Understood. That makes perfect sense and seems like an elegant solution.